cmd/srcd/daemon/daemon.go

```diff
@@ -66,6 +66,18 @@ func Kill() error {
 	return nil
 }
 
+// CleanUp removes all resources created by daemon on host
+func CleanUp() error {
+	datadir, err := datadir()
+	if err != nil {
+		return err
+	}
+
+	gitbaseIndexDir := filepath.Join(datadir, "gitbase")
```

Comment:
I wonder if this will work on windows. `datadir()` uses `/` -`filepath.ToSlash(filepath.Join(homedir, ".srcd"))`-, then here we join with the normal windows separator `\`.

Reply:
true. We should do `filepath.ToSlash` only inside daemon even if it works. Thanks!

Reply:
True that this should be tested on windows.

> `datadir()` uses `/`

Are we sure about this? Isn't `homedir.Dir()` inside `datadir()` returning a path separated by `\`?

@smacker what do you mean with "even if it works"?

Reply:
Ok, I got it now. The code this thread was pointing was the old one 😅